server/src/main/java/keywhiz/service/config/ClientAuthConfig.java

```diff
@@ -28,13 +28,23 @@
 public abstract class ClientAuthConfig {
   @JsonCreator public static ClientAuthConfig of(
       @JsonProperty("xfcc") List<XfccSourceConfig> sourceConfigs,
-      @JsonProperty("type") ClientAuthTypeConfig typeConfig) {
-    return new AutoValue_ClientAuthConfig(sourceConfigs, typeConfig);
+      @JsonProperty("type") ClientAuthTypeConfig typeConfig,
+      @JsonProperty("createMissingClients") boolean createMissingClients) {
```

Comment:
Unfortunately, Jackson does not support providing a default value for missing properties--although the [JsonProperty](https://fasterxml.github.io/jackson-annotations/javadoc/2.5/com/fasterxml/jackson/annotation/JsonProperty.html) annotation has a "defaultValue" field, this is **only for documentation** and does not affect how an empty field is interpreted.